cmd/traffic_crashlog/traffic_crashlog.cc

```diff
@@ -85,6 +85,49 @@ crashlog_open(const char *path)
   return (fd == -1) ? nullptr : fdopen(fd, "w");
 }
 
+static unsigned
+max_passwd_size()
+{
+#if defined(_SC_GETPW_R_SIZE_MAX)
+  long val = sysconf(_SC_GETPW_R_SIZE_MAX);
+  if (val > 0) {
+    return (unsigned)val;
+  }
+#endif
+
+  return 4096;
+}
+
+static void
+drop_to_nobody()
+{
+  struct passwd *pwd;
+  struct passwd pbuf;
+  char user[] = "nobody";
```

Comment:
Shouldn't this be using the configured user name from proxy.config.admin.user_id?  Which defaults to nobody.